.github/workflows/vim-patches.yml

```
@@ -24,6 +25,10 @@ jobs:
           path: ${{ env.VIM_SOURCE_DIR }}
           fetch-depth: 0
 
+      # Install libfuse-dev to automatically pull in the ABI-versioned shared
+      # library without needing to know its name
+      - run: sudo apt-get install libfuse-dev
```

```suggestion
      - run: sudo apt-get install libfuse2
```
I think that's enough -- it's what I used for nvim-treesitter: https://github.com/nvim-treesitter/nvim-treesitter/commit/cc28f563982bc916042c6c116bc32fe2b87ae016

---

It would be, but as the comment states, I explicitly went with the -dev package so this doesn't need to be updated if the ABI changes.  Maybe that's being overly defensive, though.